src/less/star-rating-select/base/star-rating-select.less

```diff
@@ -40,32 +40,27 @@ input.star-rating-select__radio_control[type="radio"] {
     z-index: 1;
 }
 
-input.star-rating-select__radio_control[type="radio"]:focus + span.star-rating-select__radio_icon {
+span.star-rating-select__radio input[type="radio"]:focus + span.star-rating-select__radio__icon {
     outline: 1px auto;
     .outline-color-token(radio-outline, color-text-secondary);
     outline-offset: 2px;
 }
 
-input.star-rating-select__radio_control[type="radio"]:focus:not(:focus-visible)
-    + span.star-rating-select__radio_icon {
+span.star-rating-select__radio
+    input[type="radio"]:focus:not(:focus-visible)
+    + span.star-rating-select__radio__icon {
     outline: none;
 }
 
-div.star-rating-select svg.star-rating-select__icon {
+div.star-rating-select span.star-rating-select__radio__icon svg {
     --color-star-rating-full: var(
         --color-star-rating-unfilled,
         @color-star-rating-icon-unfilled-foreground
     );
 }
 
-div.star-rating-select
-    input[type="radio"]:checked
-    + span.star-rating-select__radio_icon
-    svg.star-rating-select__icon,
-div.star-rating-select
-    input.star-rating-select_filled
-    + span.star-rating-select__radio_icon
-    svg.star-rating-select__icon {
+div.star-rating-select input[type="radio"]:checked + span.star-rating-select__radio__icon svg,
+div.star-rating-select input.star-rating-select_filled + span.star-rating-select__radio__icon svg {
```

Comment:
The selector `input.star-rating-select_filled` is also invalid BEM I think.
It should be
`input.star-rating-select--filled` I believe

Reply:
Yea, missed this one in the weeds. Good catch.

Reply:
That said, this was only used internally by `main.js` to drive the example on the Skin site.